backend/api/v1/sql_service.go

```diff
@@ -1887,7 +1636,7 @@ func validateQueryRequest(instance *store.InstanceMessage, statement string) err
 	return nil
 }
 
-func (s *SQLService) extractResourceList(ctx context.Context, engine storepb.Engine, databaseName string, statement string, instance *store.InstanceMessage) ([]base.SchemaResource, error) {
+func extractResourceList(ctx context.Context, storeInstance *store.Store, engine storepb.Engine, databaseName string, statement string, instance *store.InstanceMessage) ([]base.SchemaResource, error) {
```

Comment:
Can we move these to a different package instead?

Reply:
SG, how about moving to `/backend/plugin/query`, or any other suggestions?

Reply:
We can do it as follow-up. No blocking for now.